docs/site/Life-cycle-observer-generator.md

```diff
@@ -66,6 +66,13 @@ export class HelloObserver implements LifeCycleObserver {
   ) {}
   */
 
+  /**
+   * This method will be invoked when the application initializes
```

Comment:
It would be great to mention again that this method is called only once per application life time.